apps/music/js/metadata/album_art_cache.js

```diff
@@ -213,13 +277,15 @@ var AlbumArtCache = (function() {
         // Some audio tracks have an external file for their album art, so we
         // need to grab it from deviceStorage. This could also be an unsynced
         // picture that came from a regular file.
-        var getreq = AlbumArt.pictureStorage.get(picture.filename);
-        getreq.onsuccess = function() {
-          resolve(this.result);
-        };
-        getreq.onerror = function() {
-          reject(this.error);
-        };
+        LazyLoader.load('/js/metadata/album_art.js', function() {
```

Comment:
Since you're adding a lazy loading step here, I went looking for where you removed it. I can't find anywhere, so I'm guessing that means that this was just a bug: you were relying on db.js to load it and assuming that we'd never not load it.

Is this an issue in the 2.2 branch? Is there a bug that needs to be fixed there?